build_product

```diff
@@ -76,47 +270,49 @@ all_cmake_products=(
 	WRLINUX1019
 )
 
-if (( $# == 0 )); then
-	handle_wrong_argument
-fi
+NO_DERIVATIVES_OPTIONS=('-DSSG_SCIENTIFIC_LINUX_DERIVATIVES_ENABLED:BOOL=OFF' '-DSSG_CENTOS_DERIVATIVES_ENABLED:BOOL=OFF')
```

Comment:
If I run `./build_product rhel8` I get this warning from CMake:

```
CMake Warning:
  Manually-specified variables were not used by the project:

    SSG_SCIENTIFIC_LINUX_DERIVATIVES_ENABLED
```

I think you need to handle RHEL8 in a special way because the Scientific Linux 8 will not exist.